graphql-server/src/schemas/queries/samples.ts

```diff
@@ -182,9 +182,12 @@ export function buildSamplesQueryBody({
       latestSm[0] AS latestSm,
       COLLECT {
       	MATCH (s)-[:HAS_METADATA]->(sm:SampleMetadata)
+        WITH 
+          sm.cmoSampleName as cmoSampleName,
+          apoc.coll.sort(COLLECT(sm.importDate)) as orderedImportDates
       	RETURN ({
-      		cmoSampleName: sm.cmoSampleName,
-      		importDate: sm.importDate
+      		cmoSampleName: cmoSampleName,
+      		importDate: orderedImportDates[-1]
```

Comment:
`apoc.coll.sort` only takes one argument (a list) and doesn't, to my knowledge at least, support reversing the collection. We might need to upgrade apoc to get the version with `apoc.coll.reverse` but I think that's for another time.